Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_V2_0.h

```diff
@@ -43,6 +43,10 @@
 
 #define PS_ON_PIN                          PC13  // Power Supply Control
 
+#ifndef POWER_LOSS_PIN
+  #define POWER_LOSS_PIN                   PC12  // Power Loss Detection: PWR-DET
```

Comment:
Actually, I think you should move this to `pins_BTT_SKR_MINI_E3_common.h`. The same pin has been used on all versions of the SKR Mini E3 and MZ.

Reply:
I just verified that on boards that I own: E3 2.0 and MZ, that's why put it in safer place. But if it is the case for older revisions, then yeah, it can be moved to common place.